cfe_internal/update/update_policy.cf

```diff
@@ -95,7 +95,7 @@ bundle agent cfe_internal_update_policy
       "$(inputs_dir)/cf_promises_validated"
       comment => "Check whether a validation stamp is available for a new policy update to reduce the distributed load",
       handle => "cfe_internal_update_policy_check_valid_update",
-      copy_from => u_rcp("$(master_location)/cf_promises_validated", @(update_def.policy_servers)),
+      copy_from => u_rcp("$(master_location)/cf_promises_validated", @(update_def.policy_servers), "$(sys.policy_hub_port)"),
```

Comment:
Yes we discussed this, and agree that this would be better. But it will take some time to implement.